### PR TITLE
Some Inline Rename accessibility improvements

### PR DESCRIPTION
**Pull request by dpoeschl**
1. More descriptive text when an Inline Rename session begins to let the user know:
    - The name of the identifier being renamed
    - More options exist, and you can access them by invoking inline rename again
    - That the identifier being renamed can be edited at any time
2. Keyboard tabbing around the dashboard now skips invisible controls
3. The Apply and Cancel (X) buttons now read automation text correctly

### CODE REVIEW

**Comment by dpoeschl** [comment]
@ryzngard Can you take a look?

File: src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardViewModel.cs [hunk 1]
```diff
@@ -310,6 +310,11 @@ public bool DefaultPreviewChangesFlag
             }
         }
 
+        public string GetOriginalName()
```
Comment (ryzngard) [inline src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardViewModel.cs]: >GetOriginalName [](start = 22, length = 15)

nit: This looks more like a property than a function

Comment (dpoeschl) [reply src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardViewModel.cs]: Agreed, fixed.

**Comment by dpoeschl** [comment]
Also FYI @AbhitejJohn

File: src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardAutomationPeer.cs [hunk 2]
```diff
@@ -10,8 +10,11 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
     /// </summary>
     internal class DashboardAutomationPeer : UserControlAutomationPeer
     {
-        public DashboardAutomationPeer(UserControl owner) : base(owner)
+        private string _identifier;
```
Comment (mavasani) [inline src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardAutomationPeer.cs]: Nit: readonly?

**Comment by CyrusNajmabadi** [comment]
Can you show a picture?

**Comment by CyrusNajmabadi** [comment]
> More options exist, and you can access them by invoking inline rename again

What does this mean?  Thanks!

**Comment by dpoeschl** [comment]
@CyrusNajmabadi There are no visual changes. 

> More options exist, and you can access them by invoking inline rename again

This means that once Inline Rename has started, you can invoke the same command _again_ to put keyboard focus on the dashboard. From there, you can tab around to the checkboxes etc. I'm open to better wording. :)

**Comment by CyrusNajmabadi** [comment]
> This means that once Inline Rename has started, you can invoke the same command again (F2 or Ctrl+R,R or whatever) to put keyboard focus on the dashboard.

I didn't know that was a thing!

**Comment by dpoeschl** [comment]
@jinujoseph Do I need any additional signoff here?

**Comment by jinujoseph** [comment]
@dpoeschl no, pls goahead and merge

**Pull request merged**